generators/bdk/java/index.js

```diff
@@ -74,14 +74,15 @@ module.exports = class extends Generator {
         this.answers.username = this.options.username;
 
         // get latest BDK BOM version
-        try {
-            const mavenResponse = await axios.get(MAVEN_BDK_BOM_SEARCH_BASE + 'symphony-bdk-bom');
-            this.answers.bdkBomVersion = mavenResponse.data['response']['docs'][0]['latestVersion'];
-            this.log('Latest BDK version is '.green.bold + `${this.answers.bdkBomVersion}`.white.bold);
-        } catch (error) {
-            this.log(`\u26A0 Cannot retrieve latest BDK version from Maven Central. Default: ${BDK_VERSION_DEFAULT}`.grey);
-            this.answers.bdkBomVersion = BDK_VERSION_DEFAULT;
-        }
+//         try {
```

Comment:
Can you remove this comment lines?

Reply:
I'll do it later, as this PR is a simple cherry-pick from #76